pom.xml

```diff
@@ -146,7 +146,11 @@
     <jetty.version>9.4.51.v20230217</jetty.version>
     <jakartaservlet.version>4.0.3</jakartaservlet.version>
     <chill.version>0.10.0</chill.version>
-    <ivy.version>2.5.2</ivy.version>
+    <!--
+      SPARK-44968: don't upgrade Ivy to version 2.5.2 until the test aborted of
+      `HiveExternalCatalogVersionsSuite` in Java 11/17 daily tests is resolved.
```

Comment:
Thank you for adding this.

Reply:
IIUC, Java 21 works right?

Reply:
Yes, according to the test results of GA, using Java 8 and 21 works normally, but using Java 11 and 17 will fail.